tests/unit/disco/containers/TestDiscoPane.js

```diff
@@ -203,12 +213,15 @@ describe(__filename, () => {
     it('gets discovery results when results are empty', () => {
       const dispatch = sinon.stub();
       const errorHandler = new ErrorHandler({ id: 'some-id', dispatch });
-      const props = helpers.mapStateToProps(loadDiscoResultsIntoState([]));
+      const props = helpers.mapStateToProps(
+        loadDiscoResultsIntoState([]), ownProps);
 
-      render({ errorHandler, dispatch, ...props });
+      render({ errorHandler, dispatch, platform: 'Darwin', ...props });
```

Comment:
Why is this needed if `props` is constructed using `ownProps` before?

Reply:
Oh, possibly a holdover. I think it can be removed.